otx/algorithms/classification/adapters/mmcls/utils/builder.py

```diff
@@ -31,6 +35,9 @@ def build_classifier(
 
     model_cfg = deepcopy(config.model)
     model = origin_build_classifier(model_cfg)
+    logger.setLevel("WARNING")
+    model.init_weights()
```

Comment:
What is the difference between `init_weight` and `load_checkpoint` (line 45)?

Reply:
`init_weight` calls initialization methods like `Normal init` or `Linear init`. `load_checkpoint` loads incoming checkpoint from Imagenet or `load-from` argument.